scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -1614,6 +1634,100 @@ cdef class NumericalInversePolynomial(Method):
         return UError(max_error, mae)
 
 
+    def qrvs(self, size=None, d=None, qmc_engine=None):
```

Comment:
This is indeed copy-pasted verbatim (aside from removing undesirable end of line whitespace)